src/replaceAroundStep.ts

```diff
@@ -25,6 +28,83 @@ export function suggestReplaceAroundStep(
   prevSteps: Step[],
   suggestionId: number,
 ) {
+  // This detects and handles changes from `setNodeMarkup`
+  // (https://github.com/handlewithcarecollective/prosemirror-suggest-changes/issues/7)
+  if (
```

Comment:
Let's pull this out into a `suggestSetNodeMarkup` function in this module, just so that it's easier to follow